<launch>
	
	<!--<arg name="environment_1" value="environments/environment_1" />-->

	<rosparam command="load" file="$(find openaggdl)/launch/lion.v.03.yaml" />
	<rosparam command="load" file="$(find openaggdl)/launch/lion_var_types.v.03.yaml" ns="var_types"/>

	<group ns="environments/astrid">
		
		<node 
			pkg="openaggdl" 
			type="topic_filter.py" 
			name="topic_filter_1"/>

		<node 
			pkg="openaggdl" 
			type="sensor_htu21df.py" 
			name="htu21df_temp_1" >
				<param name="rate_hz" value="60" type="int"/>
		</node>

		<node 
			pkg="openaggdl" 
			type="sensor_tsl2561.py" 
			name="tsl2561_light_1" >
				<param name="rate_hz" value="1" type="int"/>
		</node>

	</group>

</launch>